src/blockdata/script.rs

```diff
@@ -400,23 +400,31 @@ impl Script {
                                opcodes::All::from(self.0[0]).classify() == opcodes::Class::IllegalOp)
     }
 
-    /// The minimum value an output to a witness script must have in order to be
+    /// The minimum value an output with a P2WPKH script_pubkey must have in order to be
     /// broadcastable on today's bitcoin network.
-    pub const WITNESS_OUTPUT_DUST_THRESHOLD: u64 = 294;
+    pub const P2WPKH_OUTPUT_DUST_THRESHOLD: u64 = 294;
```

Comment:
Are these constants actually still all that useful now. If we fell for it someone else might do the same and just use one of the constants, not paying attention to the script type :grimacing:

Reply:
Probably not, admittedly, I'm happy to remove them?

Reply:
Do if you want, I don't have too strong feelings either way. It just seemed somewhat redundant and I wanted to bring up the question.

Reply:
I do, in fact. Pushed.